base/record_replay.h

```diff
@@ -72,6 +73,7 @@ struct AutoPassThroughEvents {
 
 struct AutoDisallowEvents {
   AutoDisallowEvents() { BeginDisallowEvents(); }
```

Comment:
Can we remove this constructor?  It seems like we'll always want to have a label.